packages/ui/src/TextField.tsx

```diff
@@ -192,7 +192,17 @@ export function TextField({
   let displayValue = value;
   if (displayValue) {
     if (type === "date") {
-      displayValue = moment(value).format("MM/DD/YYYY");
+      // We get off by one errors because UTC midnight might be yesterday. So we add the timezone offset.
```

Comment:
You'd think we could have like Date and DateTime as separate things...but alas, this is probably the best way to go about it.

Reply:
Ugh right???? I'm surprised ISO doesn't have a standard way to do this.